### ISSUE TITLE
How do you calculate Mean average precision?

### ISSUE THREAD

**Issue by malarsaravanan1991** [google/next-prediction#21]
Hey hi,
I have a question regarding the evaluation metric.
How do you calculate mean average precision for activity prediction? Is it same as pascal VOC mAP 
evaluation metric or some other technique?
I see in the code that score and labels are only considered. What about the bounding boxes?

**Comment by JunweiLiang** [comment]
The activity prediction mAP is computed over each person. There is no bounding box prediction. Given the observations of a person, the model outputs a multi-class probability of all actions, which is used to compare with the ground truth. See problem formulation in Section 3 of the paper. mAP is originally proposed in the Information Retrieval field. See the definition [here](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.average_precision_score.html).
